### DISCUSSION TITLE
Random display events

### DISCUSSION THREAD

**Discussion by ebussinetz** [liedekef/events-made-easy#655] [Q&A]
I have displayed the next appointment in the calendar on the homepage of a website. Now there are several events on a certain day and I wanted to limit this to one line. Is it possible to display the events on this day in random rotation?
Thank you!

**Comment by liedekef** [comment]
Showing multiple events on 1 line is possible by changing the calendar event format settings.
If you mean to only show 1 event on a day: that is not possiblle via the calendar (it is not meant for that of course), but you can play around with the eme_events shortcode and the scope+limit arguments.
If wanting to show something in random order, keep a list of events and use jquery to randomize the list.
